tests/conftest.py

```diff
@@ -26,9 +26,9 @@ def project_main_module() -> types.ModuleType:
     """
     try:
         # This should be the project's main package; downstream projects must update this.
```

Comment:
```suggestion
```
Presumably this comment only applies to starbase and should be removed.

Reply:
Good catch!